internal/cmd/request.go

```diff
@@ -105,8 +105,13 @@ func PrintCMDResponseFull(request any, response core.APIResponse, responseType a
 		fmt.Printf("\nAutograder Response:\n---\n%s\n---\n", util.MustToJSONIndent(response))
 	}
 
+	customOutput := ""
 	if customPrintFunc != nil {
-		fmt.Println(customPrintFunc(response))
+		customOutput = customPrintFunc(response)
```

Comment:
What if empty string is valid output?
Usually better to return an error with this.